packages/Sandblocks-Smalltalk/SBVariant.class.st

```diff
@@ -3,7 +3,8 @@ Class {
 	#superclass : #SBStSubstitution,
```

Comment:
noticed late but maybe we want to move this one to -Babylonian as well?

Reply:
it's in the same category as toggledcode - and even though toggledcode might become obsolete in the future, the thought of different code paths does not relate to live programming at first glance imo